pkg/server/service.go

```diff
@@ -82,18 +85,37 @@ type criContainerdService struct {
 	client *containerd.Client
 	// streamServer is the streaming server serves container streaming request.
 	streamServer streaming.Server
+	// cgroup in which the cri-containerd is placed in
+	cgroup cgroups.Cgroup
```

Comment:
Seems like that we only need the path (as the backup for parent cgroup). Can we only save the path?